### PR TITLE
changed administrator to moderator for consistency

### PR DESCRIPTION
**Pull request by saidsellai**
GCI task

### CODE REVIEW

**Comment by anshuman73** [comment]
@brainzbot whitelist

File: critiquebrainz/frontend/messages.pot [hunk 1]
```diff
@@ -211,7 +211,7 @@ msgid "Please sign in to access this page."
 msgstr ""
 
 #: critiquebrainz/frontend/login/__init__.py:46
-msgid "You must be an administrator to view this page."
+msgid "You must be a moderator to view this page."
```
Comment (ferbncode) [inline critiquebrainz/frontend/messages.pot]: You don't need to change the `.pot` file. See http://babel.pocoo.org/en/latest/cmdline.html. CB uses `pybabel` for compiling translations. Make the changes just in the python/html code.

File: critiquebrainz/frontend/messages.pot [hunk 2]
```diff
@@ -1848,7 +1848,7 @@ msgid "Can't find a review with the specified ID."
 msgstr ""
 
 #: critiquebrainz/frontend/views/review.py:75
-msgid "Review has been hidden. You need to be an administrator to view it."
+msgid "Review has been hidden. You need to be a moderator to view it."
```
Comment (ferbncode) [inline critiquebrainz/frontend/messages.pot]: The `.mo` files are generated via the `pybabel compile` command. You may wanna just edit the code in a separate commit (the .py and the jinja files (if any)) and then use `pybabel` commands to generate the `messages.pot`(using `pybabel extract` command) and the `messages.po` (using `pybabel compile` command) files. Then add the changed files in a separate commit.

**Comment by ferbncode** [comment]
I think that the project maintainer can only build all `translations`. So, it's okay if you don't modify the `.pot` and `.po` files. Also, please push the changes to this pull request rather than opening a new one. Read more about updating pull requests here: https://stackoverflow.com/questions/9790448/how-to-update-a-pull-request-from-forked-repo. Also, please follow guidelines here while creating a pull request: https://github.com/metabrainz/guidelines/blob/master/GitHub.md

**Comment by mwiencek** [comment]
@brainzbot test this please

**Pull request closed without merging**